.cspell.json

```diff
@@ -139,8 +141,8 @@
     "Vite",
     "vstack",
     "walkthroughs",
-    "whatsapp",
     "WCAG",
+    "whatsapp",
```

Comment:
Not sure to understand the link between all the modifications in this file and the rest if the PR 🤔

Reply:
It was an automatic reordering when I've added some words inside, I could open a PR for this but it might means some confilcts to tackle afterwards.